test/unit_Zonotope.jl

```diff
@@ -160,6 +160,10 @@ for N in [Float64, Rational{Int}, Float32]
     Z = Zonotope(N[0, 0], N[1 1; -1 1])
     Z1, Z2 = split(Z, 1) # in this case the splitting is exact
     @test Z1 ⊆ Z && Z2 ⊆ Z
+    Z1, Z2, Z3, Z4 = split(Z, [1, 2], [1, 1])
+    @test Z1 ⊆ Z && Z2 ⊆ Z && Z3 ⊆ Z && Z4 ⊆ Z
+    Z = Zonotope(SVector{2}(N[0, 0]), SMatrix{2, 2}(N[1 1; -1 1]))
+    @test Z1 ⊆ Z && Z2 ⊆ Z
```

Comment:
I guess you wanted to split `Z` as well?

Reply:
Yes, I forgot!